charts/das/templates/statefulset.yaml

```diff
@@ -34,6 +34,24 @@ spec:
       priorityClassName: {{ .Values.priorityClassName }}
       {{- end }}
       initContainers:
+      {{- if index .Values "configmap" "data" "data-availability" "rest-aggregator" "sync-to-storage" "state-dir"}}
```

Comment:
This is dependent on the local storage pvc, so maybe we should add a condition for that as well.